crates/consensus/src/rounds/preproposal_wait_trigger.rs

```diff
@@ -24,9 +24,9 @@ const ETH_BLOCK_TIME: Duration = Duration::from_secs(12);
 /// The amount of the difference we scale by to reach
 const SCALING_REM_ADJUSTMENT: u32 = 3;
 /// Minimum wait time before consensus starts (in seconds)
-const MIN_WAIT_DURATION: f64 = 6.0;
+const MIN_WAIT_DURATION: f64 = 0.5;
 /// Maximum wait time before consensus starts (in seconds)
-const MAX_WAIT_DURATION: f64 = 10.0;
+const MAX_WAIT_DURATION: f64 = 2.0;
```

Comment:
this is flipped, Given that our guestimate gave us 1305ms on low and 1905ms on high. the bounds should be `MAX_WAIT=ETH_BLOCK_TIME - 1305ms`
`LOW_WAIT=ETH_BLOCK_TIME - 2000ms`
this is because this it the time in which consensus will start (ignoring order propagation) and still have enough time to complete before the block. Ideally we want to be able to push this start time as far back as possible as it allows more time for orders to accumulate and makes it so that Market Makers can price better